app/controllers/api/v0/persons_controller.rb

```diff
@@ -29,9 +33,9 @@ def competitions
     render json: person.competitions
   end
 
-  private def person_to_json(person)
+  private def person_to_json(person, private_attributes = [])
     {
-      person: person.serializable_hash(only: [:wca_id, :name, :url, :gender, :country_iso2, :delegate_status, :teams, :avatar]),
+      person: person.serializable_hash(only: [:wca_id, :name, :url, :gender, :country_iso2, :delegate_status, :teams, :avatar, private_attributes].flatten),
```

Comment:
I think this should be `person.serializable_hash(only: [...], private_attributes: [...])` so that the existing code in `person.rb` can pick it up.